core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -6,8 +6,7 @@
             class="oppia-opportunities-tab-item"
             ng-repeat="(tabName, tabDetails) in $ctrl.tabsDetails"
             ng-if="tabDetails.enabled"
-            aria-label="<[tabDetails.ariaLabel]>"
-            role="button">
```

Comment:
Why remove this? Is there a more accurate ARIA role that can be applied here (and more generally for list items)?

Reply:
No, I don't think we need to use list and listitem for ul and li elements. In fact in the guidelines you linked. It mentions we should only use it when we have too.. Like in cases like this..

```
<section role="list">
  <div role="listitem">List item 1</div>
  <div role="listitem">List item 2</div>
  <div role="listitem">List item 3</div>
</section>
```

But it would prefer it if we rewrote them using the the approritate semantic elements e.g `<ul>` and `<ol>`

```
<ul>
  <li>List item 1</li>
  <li>List item 2</li>
  <li>List item 3</li>
</ul>
```

Reply:
sg, thanks for checking!